app/lib/frontend/templates/package.dart

```diff
@@ -309,9 +309,12 @@ List<Tab> _pkgTabs(
     final exampleFilename = selectedVersion.example.filename;
     renderedExample = renderFile(selectedVersion.example, baseUrl);
     if (renderedExample != null) {
-      renderedExample = '<p style="font-family: monospace">'
-          '<b>${htmlEscape.convert(exampleFilename)}</b>'
-          '</p>\n'
+      final url = getRepositoryUrl(baseUrl, exampleFilename);
+      final escapedName = htmlEscape.convert(exampleFilename);
+      final link = url == null
+          ? escapedName
+          : '<a href="$url" target="_blank" rel="noopener noreferrer nofollow">$escapedName</a>';
+      renderedExample = '<p style="font-family: monospace"><b>$link</b></p>\n'
```

Comment:
Side question: why don't we style the link with css?

Reply:
It is a good PR to do :)